processor/tailsamplingprocessor/internal/sampling/string_tag_filter.go

```diff
@@ -41,7 +41,14 @@ var _ PolicyEvaluator = (*stringAttributeFilter)(nil)
 
 // NewStringAttributeFilter creates a policy evaluator that samples all traces with
 // the given attribute in the given numeric range.
-func NewStringAttributeFilter(logger *zap.Logger, key string, values []string, regexMatchEnabled bool, evictSize int) PolicyEvaluator {
+func NewStringAttributeFilter(logger *zap.Logger, key string, values []string, regexMatchEnabled bool, evictSize int, invertMatch bool) PolicyEvaluator {
```

Comment:
Would it make sense to wrap this in another function, making this function here just invert the result?

Reply:
Yes, it does make sense. However, as far as I could understand, there is a LRU Cache checking that must not be inverted twice. Unfortunately, it will if you wrap this function as it is now. Am I right?

Reply:
Might be -- can you point me to the code you have in mind? Or an existing test?